docs/Allocene/script/games.js

```diff
@@ -0,0 +1,189 @@
+async function createCardElement(game) {
```

Comment:
Why this function is `async`?

Reply:
When you call fetch(url, options), this operation is asynchronous, and the program does not wait for it to complete, but continues with the next instructions. Thus, for correct processing of received data and creation of game cards, the createCardElement function must also be asynchronous.

Reply:
But you call `fetch` not inside of this function, but inside of `renderCards`
So there is no any async operation in this function `createCardElement`
Please remove `async`

Reply:
Done